tools/cli/commands/create.py

```diff
@@ -447,9 +447,56 @@ def create_repo(args, gcloud_repos, repo_name):
     Raises:
       subprocess.CalledProcessError: If the `gcloud` command fails
     """
+    if utils.print_info_messages(args):
+        print('Creating repo {0}'.format(repo_name))
+
     create_cmd = ['create', repo_name]
     utils.call_gcloud_quietly(args, gcloud_repos, create_cmd)
 
+    # Clone the repo into a local scratch directory
+    datalab_dir = tempfile.mkdtemp(prefix='datalab-')
+    clone_cmd = ['clone', repo_name, datalab_dir]
+    utils.call_gcloud_quietly(args, gcloud_repos, clone_cmd)
+
+    # Add a couple of files
+    if utils.print_info_messages(args):
```

Comment:
I think the first message about creating the repo deserves to be 'info' level, but the rest of them feel more like debugging messages than info.

Consider using `if args.verbosity == 'debug':` instead of `if utils.print_info_messages(args):`